src/components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+const Calculator = () => (
+  <div className="container flex">
+    <div className="Output-bar flex button-container"> 0</div>
+    <div className="buttons">
+      <div className="col-1 button-container flex">
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
```

Comment:
- [ ] In accordance to the project requirement, please kindly implement a child for this component. It contributes to the content and structure of the parent component, making it easier to manage and maintain the codebase. This [tutorial can be helpful](https://react.dev/reference/react/Children). 👍